sync/class.jetpack-sync-client.php

```diff
@@ -684,12 +734,14 @@ private function maybe_sync_constants() {
 				 * @param mixed The value of the constant
 				 */
 				do_action( 'jetpack_sync_constant', $name, $value );
-				update_option( self::CONSTANTS_CHECKSUM_OPTION_NAME . "_$name", $checksum );
+				$constants_checksums[ $name ] = $checksum;
 			}
 		}
-	}
 
-	private function get_all_constants() {
+		update_option( self::CONSTANTS_CHECKSUM_OPTION_NAME, $constants_checksums );
+	}
+	// public so that we don't have to store an optio for each constant
```

Comment:
There is a typo in this comment. "optio" should be "option".

Also, there should probably be an empty newline above this comment.